extensions/funqy/funqy-amazon-lambda/maven-archetype/src/main/resources/archetype-resources/sam.jvm.yaml

```diff
@@ -34,8 +34,8 @@ AWSTemplateFormatVersion: '2010-09-09'
       Type: AWS::Serverless::Function
       Properties:
         Handler: io.quarkus.funqy.lambda.FunqyStreamHandler::handleRequest
-        Runtime: java8
+        Runtime: java17
```

Comment:
Thanks for the PR. Do they have a `java21` runtime? Because if so, I think we should use it by default.